package.json

```diff
@@ -47,16 +47,17 @@
   },
   "keywords": [],
   "dependencies": {
+    "amqplib": "^0.5.1",
     "async": "1.5.2",
-    "express": "^4.11.2",
     "body-parser": "1.15.0",
     "dateformat": "1.0.12",
-    "amqplib": "^0.5.1",
-    "iotagent-node-lib": "https://github.com/telefonicaid/iotagent-node-lib.git#master",
+    "express": "^4.11.2",
+    "iotagent-node-lib": "https://github.com/dcalvoalonso/iotagent-node-lib.git#task/exportConfigNgsi2",
+    "iso8601-validator": "^1.0.1",
```

Comment:
Is this actually used? Or it was added only for the test you mention at https://github.com/telefonicaid/iotagent-json/pull/265/files#r163170556 and should be removed?

Reply:
You are right, it was just for that test. I forgot to delete it. :(

Reply:
Fixed in 10969e0f085eab0d57d2bd42df675f732c2f7629